shell/browser/ui/win/notify_icon.h

```diff
@@ -91,7 +92,7 @@ class NotifyIcon : public TrayIcon {
   base::win::ScopedHICON icon_;
 
   // The context menu.
-  ElectronMenuModel* menu_model_ = nullptr;
+  RAW_PTR_EXCLUSION ElectronMenuModel* menu_model_ = nullptr;
```

Comment:
what caused this to be excluded ?

Reply:
🤔 I must have been hitting a bad build error on Windows - this previously wouldn't build, even after the needed variables in tray_icon.h|.cc and notify_icon.h|.cc were changed, but revisiting it, it's building fine. Probably a PEBCAK error on my part. Thanks!